ISISComputingGroup/ibex_gui
# Automatically add colon if the user forgets

### Description of work

Automatically adds a colon to the `PVPREFIX` if the user forgets to.

### To test
 - Change to a new instrument to trigger the set `PVPREFIX` dialog
 - Try setting a prefix that doesn't end in a colon. This should now work.
 - Try setting a prefix that does end in a colon. This should now work.

Fixes ISISComputingGroup/IBEX#2369

### Acceptance criteria
 - Setting the `PVPREFIX` should now work regardless of whether the user added a colon after the name

---

#### Code Review

    - Manual system tests for the functionality should be added if there are no automated tests
    - Manual system tests can be removed from the template if they are covered by suitable automated tests

### Functional Tests


### Final Steps
